internal/command/testversions/testprovider2

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [ $1 == "--version" ]; then
+    printf "3.2.1\n"
```

Comment:
You use `echo` in the other one. You should use the same one (whichever you want) in both.

Reply:
I switched to `printf` for this test, because `echo` wasn't letting me use `\n`. I'll change the other one to `printf` too